src/main/webapp/app/lecture/pdf-preview/pdf-preview.component.scss

```diff
@@ -0,0 +1,88 @@
+.pdf-container {
+    position: relative;
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(250px, 1fr));
+    gap: 30px;
+    max-height: 60vh;
+    height: 60vh;
+    overflow-y: auto;
+    border: 1px solid var(--border-color);
+    padding: 10px;
+    margin: 10px;
+    width: 95%;
+    box-shadow: 0 2px 5px var(--pdf-preview-pdf-container-shadow);
+    align-items: start;
+
+    @media (max-width: 800px) {
+        grid-template-columns: repeat(2, 1fr);
+    }
+
+    @media (max-width: 500px) {
+        grid-template-columns: 1fr;
+    }
+}
+
+.enlarged-container {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: var(--pdf-preview-enlarged-container-overlay);
+    z-index: 1050;
+
+    .btn-close {
+        position: absolute;
+        top: 10px;
+        right: 10px;
+        cursor: pointer;
+        color: var(--white);
```

Comment:
Just checking: Does this look good in both light mode and dark mode?

Reply:
Changed it to var(--bs-body-color)